src/java/org/apache/cassandra/db/streaming/ComponentContext.java

```diff
@@ -55,11 +57,14 @@ private ComponentContext(Map<Component, File> hardLinks, ComponentManifest manif
 
     public static ComponentContext create(Descriptor descriptor)
     {
+        if (!DatabaseDescriptor.getRawConfig().storage_flags.supports_hardlinks_for_entire_sstable_streaming)
```

Comment:
Why is this using getRawConfig?

Reply:
switched to `DatabaseDescriptor.supportsHardlinksForEntireSSTableStreaming()`